policy-controller/k8s/status/src/index.rs

```diff
@@ -60,9 +60,6 @@ pub struct Controller {
     updates: mpsc::Receiver<Update>,
     patch_timeout: Duration,
 
-    /// True if this policy controller is the leader — false otherwise.
-    leader: bool,
```

Comment:
This is only used within run(), so its scope can be narrowed for clarity.

Reply:
very nice.